### PR TITLE
[DeviceView] Add DeviceView Provider

### PR DESCRIPTION
**Pull request by struss**
This commit will add `DeviceViewProvider` to fill DeviceView.

ONE-vscode-DCO-1.0-Signed-off-by: struss <rrstrous@nate.com>

### CODE REVIEW

**Comment by struss** [comment]
currently, #1037 need to merge to list sdb device.

**Comment by struss** [comment]
Oh I forget about test. I will add that.

File: src/extension.ts
Comment (jyoungyun) [inline src/extension.ts:22]: Maybe redundant?

File: src/Backend/API.ts
Comment (jyoungyun) [inline src/Backend/API.ts:53]: (Q) Do you consider the timing issue? Does it happen frequently?

File: src/extension.ts
Comment (struss) [reply src/extension.ts:22]: I will remove this.

File: package.json
Comment (yunjayh) [inline package.json:175]: `under this` doesn't sound technical wording.. As toolchainView's refresh button, how about `Refresh devices`?

Comment (struss) [reply package.json:175]: Oh, I think I remove that by mistake. I will fix it.

File: src/Backend/API.ts
Comment (struss) [reply src/Backend/API.ts:53]: >Do you consider the timing issue?

I add this for timing issue. if this register API call start after activate this extension.

>Does it happen frequently?

sometimes occur. but I want to prevent this.

Comment (yunjayh) [reply src/Backend/API.ts:53]: There was a discussion from https://github.com/Samsung/ONE-vscode/pull/1101#issuecomment-1195099002. For now, we concluded by adding comment about it. 

Could you please add a todo comment like this commit? 2d5814350a9e1525a052c6593b2a8bd3ebe971a8 (maybe NOTE part isn't needed..)

File: src/Execute/DeviceViewProvider.ts
Comment (yunjayh) [inline src/Execute/DeviceViewProvider.ts:62]:
```suggestion
      public readonly label: string,
```

There are more auto-completed `labal`s. :wink:

Comment (struss) [reply src/Execute/DeviceViewProvider.ts:62]: I will fix it. thanks!

**Review by yunjayh — APPROVED** [review]
LGTM Thank you!

**Review by jyoungyun — APPROVED** [review]
LGTM

**Comment by llFreetimell** [comment]
@struss FYI, @yunjayh is out of office for a while. Will you wait until he comes back or merge with other member's review?

**Comment by struss** [comment]
> @struss FYI, @yunjayh is out of office for a while. Will you wait until he comes back or merge with other member's review?

please merge with other memver's review, I think.

**Review by llFreetimell — APPROVED** [review]

**Pull request merged**
